src/main/java/hillelJavaEE_02/greetings/GreetingComponent.java

```diff
@@ -0,0 +1,20 @@
+package hillelJavaEE_02.greetings;
+
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ThreadLocalRandom;
+
+@Data

```

Comment:
it is not data class. I would recommend remove this annotation and avoid publication of `greetingsList`